imports/client/components/NotificationCenter.tsx

```diff
@@ -91,15 +91,37 @@ const StyledGuessDetails = styled.div`
   text-align: end;
 `;
 
+const StyledGuessNotificationHeader = styled.div`
+  display: flex;
+  align-items: baseline;
+  flex: 1;
+
+  > * + * {
+    margin-left: 0.25rem;
+  }
```

Comment:
```suggestion
  gap: 0 0.25rem;
```
I believe this is equivalent and more straightforward.